webdriver-spec.html

```diff
@@ -4624,636 +4646,562 @@
+
+                <li><p>While <var>j</var> &lt; <var>longest action sequence</var>:
+                  <ol>
+                    <li><p>Let <var>parallel sources length</var> be the length of <var>actions</var>.
+
+                      <li><p>Let <var>i</var> be 0.
+
+                      <li><p>While <var>i</var> &lt; <var>parallel sources length</var>:
+
+                      <ol>
+                        <li><p>Let <var>raw action</var> be the entry at <var>i</var>
+                          in <var>action</var> for entry <var>j</var> in <var>actions</var>
+
+                        <li><p>Append <var>raw action</var> to <var>parallel actions</var>.
+                      </ol> <!-- /i loop -->
+                  </ol> <!-- /j loop -->
               </ol>
-            </li>
-          </ol>
-        </li>
-        <li>Else if <var>key</var> is an upper-case symbol:
-          <ol>
-            <li>If the Shift key is not pressed:
-            <ol>
-              <li>Generate a <code>keydown</code> event of the Shift key.
-              <li>Generate a sequence of <code>keydown</code>, <code>keypress</code> and <code>keyup</code> events with
-              <var>key</var> as
-              the character to emulate</li>
-              <li>Generate a <code>keyup</code> event of the Shift key.
-            </ol>
-          </li>
-          <li>else (The Shift key is pressed)
-            <ol>
-              <li>Generate a sequence of <code>keydown</code>, <code>keypress</code> and <code>keyup</code> events with
-              <var>key</var> as the character to emulate</li>
-            </ol>
-          </li>
-        </ol>
-      </li>
-      <li>Else if <var>key</var> represents a modifier key:
-        <ol>
-          <li>let <var>modifier</var> be the modifier key represented by
-          <var>key</var></li>
-          <li>If <var>modifier</var> is currently held down:
-        <ol>
-      <li>Generate a <code>keyup</code> event of <var>modifier</var></li>
-    </ol>
-  </li>
-  <li>Else:
+
+            <li><p><a>Dispatch commands</a> with argument <var>action</var>.
+          </ol><!-- /k loop -->
+
+        <li><p>Return <a>success</a> with data null.
+      </ol>
+    </section>
+    <section>
+    <h2>Releasing Actions</h2>
+  <table class="simple jsoncommand">
+    <tr>
+      <th>HTTP Method</th>
+      <th>Path Template</th>
+    </tr>
+    <tr>
+      <td>DELETE</td>
+      <td>/session/{<var>session id</var>}/actions</td>
+    </tr>
+  </table>
+
+    <p>The <dfn>Release Actions</dfn> <a>command</a> is used to cancel all current
+      <a>action chains</a>. This is the equivalent of releasing all input sources.
+    <p>The <a>remote end steps</a> are:
+      <ol>
+
+        <li><p class=issue>This needs to be completed.
+
+        <li><p>Return <a>success</a> with data null.
+      </ol>
+    </section>
+    To be able to <dfn>dispatch commands</dfn> on the remote end we need
+    to perform the following steps:
```

Comment:
_Dispatch command_ is an internal algorithm and by the looks of it we’re not passing it a JSON Object.  If we elsewhere say “dispatch command _action_”, we can just reference _action_ directly here I would think.